src/Formula/utils.ts

```diff
@@ -31,6 +31,7 @@ export function percent(value: number, info?: Info): ConstantNode<number> {
 export function infoMut(node: OptNode, info: Info): OptNode
 export function infoMut(node: NumNode, info: Info): NumNode
 export function infoMut(node: StrNode, info: Info): StrNode
+export function infoMut(node: AnyNode, info: Info): AnyNode
```

Comment:
Oops, one question. Why is this duplicated?

Reply:
The last one specifies the type for the function body. The remaining ones (including the duplicate) specify the possible type combinations. Without the duplicate, `infoMut` won't accept the `AnyNode` variant.